### ISSUE TITLE
Issue API keys to download the INI files

### ISSUE THREAD

**Issue by asgrim** [browscap/browscap-site#34]
Although so far the INI files are readily available, this is problematic and we get a huge amount of abuse.

My idea was this: in order to download the files, you must generate an API key. Generating the API key shouldn't need an account or anything, but we could probably use a Google reCaptcha or something.

The API keys should expire (maybe in 1 year or something), and probably makes sense to be a JWT as the key, then we don't even need to store anything.

Basically anything without an API key would now be blocked.

**Comment by mimmi20** [comment]
How should this work for the update function we have in the `browscap-php` library?

**Comment by asgrim** [comment]
For simplicity, I think the JWT token would just be appended to the URL, for example `http://browscap.org/stream?q=BrowsCapINI&token=<jwt_token>`. I haven't thought through it enough yet ;)
